crates/crypto/src/key_manager/mod.rs

```diff
@@ -0,0 +1,7 @@
+mod key_manager;
+pub use key_manager::*;
```

Comment:
Feels like wildcard is difficult to maintain. I typically recommend being explicit on what this is exposing. 

Tangentially, would it make sense to setup a linter? I briefly looked online and found https://github.com/rust-lang/rust-clippy

Reply:
This is pretty common in Rust. It's just saying that I can use the `key_manager` module without specifying `key_manager` again.

Without this line, caller's use statements would look like this:
`use crypto::key_manager::key_manager::KeyManager`

But with this line, callers can reference it like so:
`use crypto::key_manager::KeyManager`